solidity/contracts/converter/BancorConverter.sol

```diff
@@ -454,6 +454,34 @@ contract BancorConverter is IBancorConverter, SmartTokenController, Managed, Con
         return getFinalAmount(amount, 1);
     }
 
+
+    /**
+       @dev returns the expected return for selling the token for one of its connector tokens
+
+       @param _connectorToken  connector token contract address
+       @param _sellAmount      amount to sell (in the smart token)
+
+       @return expected sale return amount
+
+       NOTE: this not extreme precise. Use carefully.
+   */
+    function getSaleRequire(IERC20Token _connectorToken, uint256 _sellAmount)
```

Comment:
What is the difference between ```getSaleRequire``` and ```getSaleReturn```? It seems that they have the same input and return the same value. 

Why not input the eth as parameter and return the required minimum amount of smart token need to give?